app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -156,6 +161,13 @@ public void dumpAllState(Integer sessionId) {
         dumpAllState(getSession(sessionId));
     }
 
+    private boolean isLocalizedContent(@Nullable String url) {
+        return url != null && (url.equals("about:blank")
+                || url.equals(PRIVATE_BROWSING_URI)
+                || url.equals(HOME_WITHOUT_REGION_ORIGIN)
+                || url.equals(getHomeUri()));
+    }
```

Comment:
will `url.equals(HOME_WITHOUT_REGION_ORIGIN)` ever catch a case `url.equals(getHomeUri()))` does not?

Additionally, I agree with @cvan, do we really want to filter the home page since it is a web page like any other content? I don't know.

But maybe instead of filtering `about:blank`, maybe we should be filtering `url.startsWith("about:");` ?

Reply:
I have added `url.startsWith("about:") || url.startsWith("data:")` for filtering. Also adding `uri.getHost()` to check other unwanted uri.